plans/overview.md

```
@@ -0,0 +1,225 @@
+# Go Tournament Manager - Development Plan Overview
+
+## Project Vision
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)